datafusion/functions/src/core/nullif.rs

```diff
@@ -58,7 +58,7 @@ impl NullIfFunc {
         Self {
             signature:
             Signature::uniform(2, SUPPORTED_NULLIF_TYPES.to_vec(),
-                Volatility::Immutable,
+                               Volatility::Immutable,
```

Comment:
This is a strange change -- I wonder if cargo fmt doesn't see this file for some reason 🤔

Reply:
Ah, I see now https://github.com/apache/arrow-datafusion/issues/9281